letsencrypt/plugins/manual.py

```diff
@@ -57,7 +57,8 @@ class ManualAuthenticator(common.Plugin):
 cd {root}/public_html
 echo -n {validation} > {response.URI_ROOT_PATH}/{encoded_token}
 # run only once per server:
-python -c "import BaseHTTPServer, SimpleHTTPServer; \\
+$(command -v python2 || command -v python2.7 || command -v python2.6) -c \\
```

Comment:
I looked at `which`, `hash`, `type`, and `command` but `command` seemed to work best. According to [this](http://pubs.opengroup.org/onlinepubs/009696899/utilities/command.html), `command -v` also looks at aliases and functions (but not keywords). `command -v` also seemed the most portable as `type -P` doesn't work in shells such as zsh for instance.